docs/public/docs/pipeline-example-1.md

```diff
@@ -1,24 +0,0 @@
-# Transparent Elasticsearch pipeline
```

Comment:
An orphaned file - not accessible from the menu, remnant of an old documentation, info here is not up-to-date. The only way to access this file was to either know its URL or via search.